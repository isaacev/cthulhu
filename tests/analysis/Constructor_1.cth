type Foo =
  | Bar { x: Str };

#[entry]
fn main() -> () {
  Foo::Bar { y: 123 };
}
