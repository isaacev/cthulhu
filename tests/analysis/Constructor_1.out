
  TYPE MISMATCH  -------------------------------------------------------------

  Mismatch between the type `{ x: Str }` and the type `{ y: Int }`:

    : Constructor_1
  6 |   Foo::Bar { y: 123 };
    |   ^^^^^^^^^^^^^^^^^^^

